tox.cover.sh

```diff
@@ -18,7 +18,7 @@ cover () {
   if [ "$1" = "certbot" ]; then
     min=98
   elif [ "$1" = "acme" ]; then
-    min=100
+    min=99
```

Comment:
Rather than reducing coverage here, can we add a test for the line now missing coverage? This can be a UNIX only test if that's easier.

Reply:
Even better, code not covered was a patch, not needed anymore. And it allows more tests to be run untouched on Windows.

I removed the patch, and coverage is 100% again.